.travis.yml

```diff
@@ -39,6 +39,6 @@ script:
   - make -j$(nproc) check
   - sudo make install
```

Comment:
I don't think we need this "sudo make install", we can just set PATH to point to the src dir as the first search path in the test_all.sh script.